test/integration/test_recover.c

```diff
@@ -3,7 +3,7 @@
 
 /******************************************************************************
  *
- * Fixture holding a bootstrapd raft caluster.
+ * Fixture holding a bootstrapped raft cluster.
```

Comment:
I'm not entirely certain about `bootstrapped`, it could be `bootstrap`...

Reply:
bootstrapped is fine